### DISCUSSION TITLE
How to generate color gradient bar which can be shown below every simulation result?

### DISCUSSION THREAD

**Discussion by utkarshpancholi** [simnibs/simnibs#169] [2. General]
I have 10 patient data for peak electric field and focality. Every scalar mesh have its own color gradient bar representation for different quantities like norm E,  Mean  , norm J, E-normal, J-normal... 

If i have 10 subject ...can i have only one color gradient bar for 10 patients ...I wan to use only one color gradient bar instead 10 bar representation for each subject separately. 

Please help.

**Comment by oulap** [comment]
You  can set the min and max of the color bar to be the same over all subjects. In gmsh (after opening the mesh with the simulation) click on Tools -> Options. Then select the view you want to work with, foe example View[1]. After that you can set the Min and Max values of the visualization bar. 

If you set the Min and Max to  be the same over all subjects the electric field visualization will be on the same scale.
